### PR TITLE
Fixes #37756 - Add product host count to subscription - general report

### PR DESCRIPTION
**Pull request by nadjaheitmann**
Needs https://github.com/Katello/katello/pull/11091 .

### CODE REVIEW

**Comment by nadjaheitmann** [comment]
@jeremylenz This is the follow-up for the Host Product Count. Do you mind having a look here as well? The Redmine issue test fails because I referenced an issue in the Katello project.

**Comment by jeremylenz** [comment]
> The Redmine issue test fails because I referenced an issue in the Katello project.

You'll have to open a new Redmine in the Foreman project and adjust the commit message.

File: app/views/unattended/report_templates/subscription_-_general_report.erb [hunk 1]
```diff
@@ -37,7 +37,8 @@ require:
...
           'End date': pool.end_date,
           'Days Remaining': pool.days_until_expiration,
-          'Organization': pool.organization.name
+          'Organization': pool.organization.name,
+          'Product Host Count': pool.product_host_count
         ) -%>
 <%- end -%>
-<%= report_render -%>
\ No newline at end of file
+<%= report_render -%>
```
Comment (jeremylenz) [inline app/views/unattended/report_templates/subscription_-_general_report.erb:44]: No need to add a newline here, but if lint is happy I'm happy

Comment (nadjaheitmann) [reply app/views/unattended/report_templates/subscription_-_general_report.erb:44]: Not sure what is happening.

**Comment by nadjaheitmann** [comment]
> > The Redmine issue test fails because I referenced an issue in the Katello project.
> 
> You'll have to open a new Redmine in the Foreman project and adjust the commit message.

I did.

**Review by jeremylenz — APPROVED** [review]

**Comment by jeremylenz** [comment]
Thanks @nadjaheitmann ! merging

**Pull request merged**